internal/common/common.go

```diff
@@ -320,3 +316,16 @@ func VerifyCaBundle(pemCerts []byte) error {
 	}
 	return nil
 }
+
+// CountActiveHosts returns the number of non-disabled hosts for a given cluster
+func CountActiveHosts(cluster *Cluster) int {
```

Comment:
try using cluster.ReadyHostCount instead. it should already contain this value.

Reply:
done